gooey-gui/app/components/BulkProgressCard.css

```
@@ -0,0 +1,567 @@
+  -webkit-line-clamp: 1;
+  line-clamp: 1;
+  flex: 1 1 auto;
+  max-width: 100%;
+  min-width: 0;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  text-decoration: underline;
+  text-underline-offset: 0.12em;
+  vertical-align: bottom;
+  white-space: normal;
+  word-break: break-word;
```

Stylelint (17.12.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)